das/daser.go

```diff
@@ -77,6 +78,8 @@ func (d *DASer) sampling(ctx context.Context, sub header.Subscription) {
 			log.Errorw("DASer failed to get next header", "err", err)
 		}
 
+		startTime := time.Now()
+
 		err = d.da.SharesAvailable(ctx, h.DAH)
```

Comment:
This is incorrect because it includes the time waiting for a new Header

Reply:
?

Reply:
Fixed with b20bc81?